jetty9-compat/src/main/docker/Dockerfile

```diff
@@ -12,7 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM jetty9:@@image.label@@
+FROM jetty9:${docker.label.prefix}${jetty.version}
+
+# Create env vars to identify image
+ENV GAE_IMAGE_NAME ${project.artifactId}
+ENV GAE_IMAGE_LABEL ${docker.label.prefix}8-jre
```

Comment:
I still don't see why GAE_IMAGE_LABEL for jetty9-compat is ${docker.label.prefix}**8-jre** but we can talk about it in our meeting soon.